python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -41,4 +42,16 @@ export default memo(function RunPromptButton({
       )}
     </Button>
   );
+
+  return !disabled ? (
+    button
+  ) : (
+    <Tooltip
+      label="Can't run while another prompt is running :("
```

Comment:
```suggestion
      label="Can't run while another prompt is running"
```
Let's remove the :( lol